### ISSUE TITLE
Support <title>: <file.md|directory> a la nav in arrange?

### ISSUE THREAD

**Issue by vicaya** [lukasgeiter/mkdocs-awesome-pages-plugin#14]
`.pages.arrange` is a convenient layout overriding mechanism. It'd be great if you can make it work a little more like the nav directive. e.g.
```
arrange:
  - Home: README.md
  - Background: background-materials-directory
```
The first form (`<title>: <file.md>`) is identical to nav. The second form would save us from having to create a .pages file with a title directive just to override the directory title.

**Comment by andrzejwp** [comment]
+1 on this

**Comment by timvink** [comment]
+1  :).  Helps to separate content (`nav`) from config (rest of `mkdocs.yml`). Ideally I would just copy/paste the `nav` section to `.pages`

**Comment by AshneetG** [comment]
+1

**Comment by lukasgeiter** [comment]
Hi everyone! I've just released v2.3.0 which brings the `nav` attribute to the `.pages` file. Like you've suggested, it works the same way as the one from `mkdocs.yml`, allowing you to specify titles, add links and more. [See the updated README for details](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin#customize-navigation).

**Comment by timvink** [comment]
Awesome, thanks!
